airflow/utils/db_cleanup.py

```diff
@@ -115,6 +116,9 @@ def readable_config(self):
     _TableConfig(table_name="celery_tasksetmeta", recency_column_name="date_done"),
 ]
 
+if conf.get("webserver", "session_backend") == "database":
+    config_list.append(_TableConfig(table_name="session", recency_column_name="expiry"))
+
```

Comment:
Yes. We could add a new command for that, but I wanted to keep the scope of this one to be "bugfix" level - so not to add new features, in case we do not manage to get it in for 2.7.0 and would like to merge it for 2.7.1

Reply:
Added task for the dedicated clean-all-expired-sessions command https://github.com/apache/airflow/issues/33370